apps/spruce/src/pages/task/taskTabs/TaskHistory/types.ts

```diff
@@ -13,6 +13,7 @@ export type GroupedTask =
   | {
       inactiveTasks: null;
       task: TaskHistoryTask;
+      isMatching: boolean;
     };
 
```

Comment:
To avoid the awkward statements of `"isMatching" in t`, I'd rather we just add `isMatching` to both of these and just set `isMatching` to false for inactive commits.